docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+build process in AWS, which was set up to watch for branch changes during the `./deploy_stack.sh main` step.
+
+This means that all you have to do to deploy changes to an already-deployed pipeline is just push your changes to the
+`main` branch.
+
+!!! success
+
+    You've now deployed a pipeline stack to AWS and you know how to update and add new pipelines on-the-fly!
+
+## Viewing your Resources in AWS
+
+You can use the AWS UI to view the resources that were created during the build.
+
+<div class="grid cards" markdown>
+
+* :material-file-multiple:{ .lg .middle } **Code Pipeline**
+
+    ---
+
+    From here you can check the status of your code build to make sure it is running successfully.
+
+    [:material-link: Link](https://us-west-2.console.aws.amazon.com/codesuite/codepipeline/pipelines/)
+
+* :octicons-container-16:{ .lg .middle } **ECR Container Repository**
+
+    ---
+
+    From here you can check the status of your built images.
+
+    [:material-link: Link](https://us-west-2.console.aws.amazon.com/ecr/repositories?region=us-west-2>)
+
+* :material-bucket-outline:{ .lg .middle } **S3 Buckets**
+
+    ---
+
+    From here you can check the contents of your input and output buckets.
+
+    [:material-link: Link](https://s3.console.aws.amazon.com/s3/buckets?region=us-west-2>)
+
+* :simple-awslambda:{ .lg .middle } **Lambda Functions**
+
+    ---
+
+    You can see the lambda functions that were created for each pipeline here.
+
+    [:material-link: Link](https://us-west-2.console.aws.amazon.com/lambda/home?region=us-west-2#/functions>)
+
+* :material-bridge:{ .lg .middle } **Event Bridge Cron Rules**
+
+    ---
+
+    From here you can check what cron events have been set up for any cron-triggered pipelines.
+
+    [:material-link: Link](https://us-west-2.console.aws.amazon.com/events/home?region=us-west-2#/rules>)
+
+* :octicons-stack-24:{ .lg .middle } **Cloud Formation Stack**
+
+    ---
+
+    You can see the resources that were created via the CDK deploy. You can also delete the stack from here to clean up
+    those resources.  Note that any lambda functions and Event Bridge cron rules created via the CodePipeline build are
+    NOT part of the stack, so these would have to be removed by hand.
+
+    [:material-link: Link](https://us-west-2.console.aws.amazon.com/cloudformation/home?region=us-west-2#/stacks?filteringText=&filteringStatus=active&viewNested=true>)
+
+</div>
+
+## Removing your AWS Stack
+
+If for some reason you would like to completely remove everything that's been deployed, then follow the steps below for
+each branch you deployed.
+
+1. Make sure the input and output S3 buckets are completely empty.
+2. Delete the CloudFormation stack. It should be named like `pipeline-template-main-CodePipelineStack`
+3. Navigate to the Lambda UI and delete any lambda functions named like `pipeline-template-branch-lambda-*`.
```

Comment:
`pipeline-template-[BRANCH_NAME]-lambda-*`.

Reply:
Adding an annotation for this. Do you know if there is a way for us to automatically remove these lambda functions too? This seems like it could get pretty tedious if you have more than a handful of configs or if you're deploying from several branches